tools/dune_rule_gen/LICENSE

```diff
@@ -0,0 +1,23 @@
+The MIT License
```

Comment:
Why this exception? All the code is LGPL, no?

Reply:
This is derived from Dune's code, so IMHO it is better we keep the same license.

Reply:
It should also ease moving code back and forth between Dune's code and the local coq_dune.